tests/graph_components/validators/test_config_validator.py

```diff
@@ -0,0 +1,805 @@
+from typing import Dict, List, Optional, Set, Text, Any, Tuple, Type
```

Comment:
I think the following cases are currently not covered:
- missing `EntitySynonymMapper`
- entity roles / groups without DIET / CRF

Reply:
fun fact: entity roles / groups without DIET/CRF was supposed to be handled by the parametrization with "{DIET ,... }.difference(TRAINABLE_EXTRACTORS)" but the order there is obviously wrong 🤦‍♀️  

(and completely missed EntitySynonymMapper)